interface-definitions/dns-forwarding.xml.in

```diff
@@ -605,6 +605,20 @@
                 </properties>
               </leafNode>
               #include <include/listen-address.xml.i>
+              <leafNode name="listen-port">
```

Comment:
Please use `#include <include/port-number.xml.i>` instead. We try to have a consistent CLI for the same options and also re-use already existing CLI definitions (`include/port-number.xml.i`)

Reply:
I use that at first, but I don't know how to override the default value when using the include. It needs to be 53, could you tell me how to set the default value?

Reply:
Like below, taken from snmp template

```
#include <include/port-number.xml.i>
<leafNode name="port">
  <defaultValue>161</defaultValue>
 </leafNode>
```

Reply:
Thank you, I've updated the commit, PTAL.